kube_controller_manager/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Changed***:
```

Comment:
This does not appear to be breaking so it should be `Added`

Reply:
updated!